demo/src/pick_place_task.cpp

```diff
@@ -194,16 +215,24 @@ void PickPlaceTask::init() {
   ---- *               Generate Grasp Pose                *
 		 ***************************************************/
 		{
-			// Sample grasp pose
-			auto stage = std::make_unique<stages::GenerateGraspPose>("generate grasp pose");
-			stage->properties().configureInitFrom(Stage::PARENT);
-			stage->properties().set("marker_ns", "grasp_pose");
-			stage->setPreGraspPose(hand_open_pose_);
-			stage->setObject(object);
-			stage->setAngleDelta(M_PI / 12);
-			stage->setMonitoredStage(current_state_ptr);  // Hook into current state
-
-			// Compute IK
+            // The object grasp is defined by combining two stages, a GenerateGraspPose and a ComputeIK pose. The
```

Comment:
"latches onto the object" implies a connection that I don't really see. "connects" seems to make more sense.

"The generateGraspPose" should be "The generateGraspPose stage" or "...container". The phrase "a ComputeIK pose" seems to be used to mean "a ComputeIKpose stage". This is confusing.

`GenerateGraspPose` is written both with and without capital first letter.